CHANGELOG.md

```diff
@@ -154,8 +154,8 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - A bug in network map page that caused delay of telemetry log loading. #1545
 - Windows "run as a user" powershell command for manual agent runs. #1556
-- A bug where T1216_random_executable.exe was copied to disk even if the signed
-  script proxy execution PBA was disabled. #1864
+- A bug in the map where side pane would not appear if the node was
+  dragged around before click. #2914
```

Comment:
Was this bug present in the last release? Or did we introduce it since then?

Reply:
The line you changed was added in #2462.

Reply:
After being removed in #2460 first. Never mind, it was most likely always there.